Makefile

```diff
@@ -344,6 +345,19 @@ uninstall:
 	@echo "$(WHALE) $@"
 	@rm -f $(addprefix $(DESTDIR)/bin/,$(notdir $(BINARIES)))
 
+#TODO(mikebrow): refactor install-critools and install-cni to work for local host and for release
+ifeq ($(GOOS),windows)
+install-deps:
+	script/setup/install-runc
+	#script/setup/install-critools
+	#script/setup/install-cni-windows
+else
+install-deps: ## install cri dependencies
+	script/setup/install-seccomp
+	script/setup/install-runc
+	#script/setup/install-critools
+	#script/setup/install-cni
```

Comment:
Is this PR WIP?

Reply:
added another commit to close the new todo.. ready for review.